packages/@aws-cdk/aws-ec2/lib/instance-types.ts

```diff
@@ -566,6 +566,16 @@ export enum InstanceClass {
    */
   Z1D = 'z1d',
 
+  /**
+   * High memory and compute capacity instances based on Intel Xeon Scalable (Cascade Lake) processors, 2nd generation
+   */
+  HIGH_COMPUTE_MEMORY2 = 'm5zn',
```

Comment:
Hmm... 

I get that the blog post mentions this as building on the success of the z1d instances, but this does belong to the "m" family, so I think we should reflect that in the name (e.g., `STANDARD5_HIGH_COMPUTE`?) and description.

Reply:
Updated!

Reply:
Please review and lmk. TY!